src/App.tsx

```diff
@@ -1,109 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const optionsPerPage = [3, 5, 10, 20];
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(optionsPerPage[1]);
```

Comment:
Consider initializing the `perPage` state with a default value that is more indicative of a typical use-case scenario. The second option (5 items per page) is a reasonable default, but ensure this aligns with your application's requirements.